data/supporters.json

```diff
@@ -1429,6 +1429,27 @@
           "twitter": "_amatellanes"
         }
       ]
+    },
+    {
+      "name": "ThoughtWorks Tech Talks NYC Meetup",
+      "city": "New York",
+      "country": "USA",
+      "link": "https://www.meetup.com/ThoughtWorks-Tech-Talks-NYC/",
+      "twitter": "hashtag/ThoughtWorksNYC",
```

Comment:
Right now we don’t have support for hashtags and it would break our website a little. Do you maybe want to remove this line until #150 is solved?